.github/workflows/tests.yml

```diff
@@ -46,14 +46,14 @@ jobs:
         docker compose up --build -d
 
         # deploy static files and migrate database
-        docker compose run django python manage.py collectstatic --no-input
-        docker compose run django python manage.py migrate --no-input
+        docker compose exec django python manage.py collectstatic --no-input
+        docker compose exec django python manage.py migrate --no-input
 
         # sprinkle some test data (refreshing computed fields is done from the command handler)
-        docker compose run django python manage.py init --data --users
+        docker compose exec django python manage.py init --data --users
 
-    - name: Healthcheck
-      run: wget --no-check-certificate https://localhost/oapif/collections/signalo_core.pole/items
```

Comment:
If you remove this test, ensure that we still somehow cover this case (integration, incl. caddy).

Otherwise keep it, and run the django tests afterwards.